--- gst-rtsp-0.10.6/bindings/python/Makefile.am.link	2010-11-21 12:16:03.000000000 +0000
+++ gst-rtsp-0.10.6/bindings/python/Makefile.am	2010-11-21 12:16:20.000000000 +0000
@@ -23,7 +23,7 @@
 rtspserver_la_LIBADD = $(top_builddir)/gst/rtsp-server/libgstrtspserver-@GST_MAJORMINOR@.la \
 		$(GST_PLUGINS_BASE_LIBS) $(GST_BASE_LIBS) \
 		-lgstrtp-@GST_MAJORMINOR@ -lgstrtsp-@GST_MAJORMINOR@ \
-        	-lgstsdp-@GST_MAJORMINOR@ $(GST_LIBS) $(LIBM)
+        	-lgstsdp-@GST_MAJORMINOR@ $(GST_LIBS) $(LIBM) -lpython$(PYTHON_VERSION)
 rtspserver_la_SOURCES = rtspservermodule.c 
 nodist_rtspserver_la_SOURCES = rtspserver.c
 
